MaterialLib/SolidModels/Ehlers-impl.h

```diff
@@ -763,7 +763,7 @@ SolidEhlers<DisplacementDim>::integrateStress(
         std::unique_ptr<
             typename MechanicsBase<DisplacementDim>::MaterialStateVariables>{
             new StateVariables<DisplacementDim>{state}},
-        std::move(tangentStiffness)};
```

Comment:
Only a Comment: Since `KelvinMatrix` is fixed-size, there is probably no gain from moving `tangentStiffness` here. Or is there some copy elision magically taking place here?